working-groups/parallel-rustc/NOTES.md

```diff
@@ -0,0 +1,9 @@
+#### *<p align="center">This page is an example template. No meetings yet.</p>*
```

Comment:
I'd probably remove this and just say something like "This working group hasn't had any meetings yet."

Reply:
This line from the template still exists.

Reply:
Yeah, I thought it kinda makes sense to leave it like this until there are actual meeting notes.

Reply:
Ah, I hadn't noticed that it changed to remove the "this page is an example template" part of the message. My bad.